tbx/project_styleguide/templates/patterns/styleguide/components/components.html

```diff
@@ -1,7 +1,14 @@
+<h1 class="heading--one">Styleguide</h1>
 <div class="p-4">
-    <p class="underline mb-2">Avatar</p>
+    <h2 class="underline mb-2">Avatar</h2>
     {% include "patterns/atoms/avatar/avatar.html" %}
 
-    <p class="underline mt-8 mb-2">Avatar (small)</p>
+    <h2 class="underline mt-8 mb-2">Avatar (small)</h2>
     {% include "patterns/atoms/avatar/avatar.html" with classes="avatar--small" %}
+
+    <h2 class="underline mt-8 mb-2">Pullquote (person)</h2>
+    {% include "patterns/molecules/streamfield/blocks/pullquote_block.html" with value=blockquote_person_data %}
```

Comment:
It would be nice to show the cta variant here, and if possible what it looks like in a different theme?

Reply:
As discussed on Slack i've added a theme chooser here: https://github.com/torchbox/torchbox.com/pull/26/commits/ef3bbde07506c447d3d2deffad308d8bff0bf6b0

and added an example with a CTA here: https://github.com/torchbox/torchbox.com/pull/26/commits/25721e6a7077103dceeeeb3589ba85b341c9f104